src/promptflow/promptflow/batch/_python_executor_proxy.py

```diff
@@ -48,15 +53,22 @@ def _exec_batch(
         run_id: Optional[str] = None,
     ) -> List[LineResult]:
         self._flow_executor._node_concurrency = DEFAULT_CONCURRENCY_BULK
-        with self._flow_executor._run_tracker.node_log_manager:
+        if isinstance(self._flow_executor, ScriptExecutor):
```

Comment:
Add a todo to refine this, since script executor actually doesn't have the "node" concept, will this still useful? We may refine this part in future PRs.

Reply:
Added.